### DISCUSSION TITLE
How to tell podman to use the default remote connection by default?

### DISCUSSION THREAD

**Discussion by rdbisme** [containers/podman#24425] [General]
I'd like to use the same podman server used by Podman Desktop on Windows, also from within my WSL.

I have it correctly configured: 

```
❯ podman system connection list
Name        URI                                                                     Identity    Default     ReadWrite
wsl         unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-user.sock              true        true
```

but this always require to specify `--remote`. 

is there a way to default to remote by default?

**Comment by nalind** [comment]
Try setting `CONTAINER_HOST` in your environment to the URI of the connection you want to default to using.

**Reply by rdbisme** [reply]
Thanks, that works! :)
